src/shared/tests.c

```diff
@@ -71,3 +72,10 @@ const char* get_testdata_dir(const char *suffix) {
         strncpy(testdir + strlen(testdir), suffix, sizeof(testdir) - strlen(testdir) - 1);
         return testdir;
 }
+
+bool slow_tests_enabled(void) {
+        int r;
+
+        r = getenv_bool("SYSTEMD_SLOW_TESTS");
+        return r >= 0 ? r : SYSTEMD_SLOW_TESTS_DEFAULT;
```

Comment:
not that it matters, but i'd probably log here if r < 0, i.e. on parse errors...